### PR TITLE
Add more SG manipulation functions

### PR DESCRIPTION
**Pull request by anton-b**
### Problem
- Erlcloud can't authorize egress in the SGs
- Erlcloud can't revoke ingress/egress rules for EC2-VPC

### Resolution
- Add authorize_security_group_egress/2 authorize_security_group_egress/3
- Add revoke_security_group_ingress/2 revoke_security_group_ingress/3
revoke_security_group_egress/2 revoke_security_group_egress/3

### CODE REVIEW

**Comment by anton-b** [comment]
@motobob please take a look

File: src/erlcloud_ec2.erl [hunk 1]
```diff
@@ -2917,11 +2964,26 @@ reset_snapshot_attribute(SnapshotID, create_volume_permission, Config)
...
     Params = [{"GroupName", GroupName}|ingress_spec_params(IngressSpec)],
-    ec2_simple_query(Config, "RevokeSecurityGroupIngress", Params).
+    ec2_simple_query(Config, "RevokeSecurityGroupIngress", Params);
+
+revoke_security_group_ingress(GroupId, VpcIngressSpec, Config)
+  when is_list(GroupId), is_list(VpcIngressSpec) ->
+    Params = [{"GroupId", GroupId} |vpc_ingress_spec_to_params(VpcIngressSpec)],
+    ec2_simple_query(Config, "RevokeSecurityGroupIngress", Params, ?NEW_API_VERSION).
+
+-spec revoke_security_group_egress(string(), [vpc_ingress_spec()]) -> ok.
```
Comment (alexo-spb) [inline src/erlcloud_ec2.erl]: `vpc_ingress_spec()` -> `vpc_egress_spec()`

File: src/erlcloud_ec2.erl [hunk 2]
```diff
@@ -2917,11 +2964,26 @@ reset_snapshot_attribute(SnapshotID, create_volume_permission, Config)
...
+revoke_security_group_egress(GroupName, VpcEgressSpec) ->
+    revoke_security_group_egress(GroupName, VpcEgressSpec, default_config()).
+
+-spec revoke_security_group_egress(string(), [vpc_ingress_spec()], aws_config()) -> ok.
```
Comment (alexo-spb) [inline src/erlcloud_ec2.erl]: Bad type `vpc_ingress_spec()` -> `vpc_egress_spec()`

**Comment by anton-b** [comment]
@alexo-spb Fixed, please take a look

**Comment by anton-b** [comment]
@SergeyOvchinnik Please take a look as well.

**Comment by alexo-spb** [comment]
lgtm

**Comment by anton-b** [comment]
@motobob, merge please it has been approved.

**Comment by SergeyOvchinnik** [comment]
Looks good to me, +1

**Pull request merged**